core/common/src/rpc/IModelReadRpcInterface.ts

```diff
@@ -84,4 +84,6 @@ export abstract class IModelReadRpcInterface extends RpcInterface {
   public async getGeoCoordinatesFromIModelCoordinates(_iModelToken: IModelRpcProps, _props: string): Promise<GeoCoordinatesResponseProps> { return this.forward(arguments); }
   /** @beta */
   public async getGeometrySummary(_iModelToken: IModelRpcProps, _props: GeometrySummaryRequestProps): Promise<string> { return this.forward(arguments); }
+  /** @alpha */
+  public async getTextureImage(_iModelToken: IModelRpcProps, _textureName: string): Promise<Uint8Array | undefined> { return this.forward(arguments); }
```

Comment:
We discussed making this accept an "options" object, so that additional properties like supported compressed formats could be easily added in future.

Reply:
@pmconne Can RPC methods not get optional parameters later?

Reply:
They can. I've grown to hate optional arguments though. We know we will eventually want to add compressed formats, possibly max supported dimensions, and potentially other things (like, I'll take uncompressed if you don't already have a compressed version, so that I get it faster).
With an options object it's easy to add new ones and easy for callers to specify only the options they care about.
`getTextureImage({ textureId, maxSize: 4096 })` is easier to understand and to write than `getTextureImage(textureId, undefined, true, 4096)`

Reply:
Yeah, sounds reasonable.  I'll work on adding this.

Reply:
Options object as the parameter to getTextureImage has been added.